### PR TITLE
DFA-1059 - Check if the user's email is all already registered with t…

### PR DESCRIPTION
**Pull request by HarshVasudev**
- Check if the user's email is all already registered with 
- Code tidy up as per ESlint.

### CODE REVIEW

**Review by JohnWattsGDS — CHANGES_REQUESTED** [review]
We need feature tests for this flow.

You will need to alter the `StubCognitoClient`.  Line 122 in the `createUser` stub implementation should specify the method `createUser` instead of `login`; it looks like there was some copy and pasting that went wrong.

Once that's done then if the feature test tries to create and account with the email address `inuse@foo.gov.uk` the correct exception will be thrown to trigger this flow.  You can check the first entry in `stubs/stubs-config.json` to see how the `StubCongitoClient` knows to throw the error.

It should probably be a new `scenario` in `features/sign-up.feature` - the first five steps of most if not all scenarios should be copyable if you change the email and redirected-to values.  Also check that the email address is rendered on the page.

File: express/src/controllers/create-account.ts [hunk 1]
```diff
@@ -18,7 +23,7 @@ export const processGetEmailForm = async function (req: Request, res: Response,
     } catch (error) {
         if (error instanceof UsernameExistsException) {
             // TODO We need to handle this properly with another flow
```
[the remark this answers is not in the document]
Comment (HarshVasudev) [reply express/src/controllers/create-account.ts:25]: Done.

[the remark this answers is not in the document]
Comment (JohnWattsGDS) [reply express/src/controllers/create-account.ts:25]: I don't think it's been added and committed.

File: express/src/controllers/create-account.ts [hunk 2]
```diff
@@ -219,7 +223,7 @@ export const resendEmailVerificationCode = async function (req: Request, res: Re
     const emailAddress: string = req.body.emailAddress;
     const s4: SelfServiceServicesService = await req.app.get("backing-service");
 
-    let result: any;
+    let result: AdminCreateUserCommandOutput;
```
Comment (JohnWattsGDS) [inline express/src/controllers/create-account.ts:226]: 😄 👍🏻

**Review by JohnWattsGDS — APPROVED** [review]
👍🏻

**Pull request merged**